src/Mautic/Form.php

```diff
@@ -41,7 +41,7 @@ public function __construct(Mautic $mautic, $id)
      *
```

Comment:
Update the doc bock too, please

Reply:
Commit 5f770b0a9522cf1c1d8d94f438e0afca36347b9a updates the doc block. I'm not sure, if its done the right way.